.eslintignore

```diff
@@ -1,6 +1,7 @@
 # Libs we don't need to check
 node_modules
 **/vendor
+lib
```

Comment:
Sorry, forgot to mention except for 

  45:13  error  Don't use process.exit(); throw an error instead  no-process-exit
  75:5   error  Don't use process.exit(); throw an error instead  no-process-exit
  82:3   error  Don't use process.exit(); throw an error instead  no-process-exit